File: Lab 4/t2.c
#include <stdio.h>

int main (void)
{
  int a = 0, b = 0, c = 0;
  puts ("Please input three integers:");
  scanf ("%d %d %d", &a, &b, &c);
  printf (
    "Among %d, %d and %d, %d is the greatest.\n",
    a, b, c,
    a > c ? (a > b ? a : b) : (b > c ? b : c)
  );
  return 0;
}

File: Lab 4/t2.go
package main

import "fmt"

func main() {
  var a, b, c int = 0, 0 ,0
  fmt.Println("Please input three integers:")
  fmt.Scanf("%d %d %d", &a, &b, &c)
  fmt.Printf("Among %d, %d and %d, %d is the greatest.\n", a, b, c, tri_bigger(a, b, c))
}

func tri_bigger(a, b, c int) int {
  if a > c {
    if a > b {
      return a
    } else {
      return b
    }
  } else {
    if b > c {
      return b
    } else {
      return c
    }
  }
}
